repositories/src/commonMain/kotlin/RepositoriesAPI.kt

```diff
@@ -573,4 +573,33 @@ public value class RepositoriesAPI(public val gitHubClient: GitHubClientImpl) {
             }
         }
     }
+
+    /**
+     * Lists public repositories for the specified user.
+     * Note: For GitHub AE, this endpoint will list internal repositories for the specified user.
+     *
+     * Represents [this endpoint](https://docs.github.com/en/rest/reference/repos#list-repositories-for-a-user).
+     *
+     * @param username username of user to list repositories
+     * @param builder builder for configuring the request
+     * @return [List] of repositories
+     */
+    public suspend fun listRepositoriesForUser(
+        username: String,
+        builder: ListRepositoriesForUserRequestBuilder.() -> Unit
+    ): List<Repository> {
```

Comment:
```suggestion
    ): Flow<Repository> {
```
Should return a Flow